include/nbl/builtin/hlsl/bxdf/common.hlsl

```diff
@@ -467,7 +487,7 @@ struct SIsotropicMicrofacetCache
         LplusV_rcpLen = rsqrt<scalar_type>(2.0 + 2.0 * VdotL);
 
         this_t retval;
-
+        retval.VdotL = VdotL;
         retval.VdotH = LplusV_rcpLen * VdotL + LplusV_rcpLen;
         retval.LdotH = retval.VdotH;
         retval.NdotH = (NdotL + NdotV) * LplusV_rcpLen;
```

Comment:
doesn't some bit of computeMicrofacetNormal do the same math for you?

btw if you look closely you can notice that `VdotH` and `LdotH` will always be positive. 

But `NdotH` can be negative (and it should be absolute),

This needs to be rectified, flip sign on all dot products with `H` if `NdotL+NdotV<0`

Reply:
continued in https://github.com/Devsh-Graphics-Programming/Nabla/pull/916/files#r2310049891